### DISCUSSION TITLE
Tidal on Sonos

### DISCUSSION THREAD

**Discussion by Zjoskuh** [tidal-music/discussions#95] [Q&A]
Hi, I'm Jos ... and I don't know if I'm in the right place but I'm here via Tidal support. I'm not a developer ... 
I don't know anything about it. 
But I have a question: I use Tidal on my Sonos system and in the Tidal app my favorite artists are neatly in alphabetical order, but in the Sonos app they are in order of addition ... so the latest is at the top. According to Sonos it's due to the 'API' of Tidal ... Tidal says it's due to the Sonos app. 
Is there a simple way to solve this? If it's not possible then too bad. 
Thanks in advance.

Kind regards,
Jos

**Comment by stefanmoro** [comment]
Hey @Zjoskuh ,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to an existing product offering, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks

**Reply by Zjoskuh** [reply]
Thanks for answering my question… I understand I’m in the wrong place. 
But I’m being sent from Tidal to Sonos, from Sonos back to Tidal Support and then from Tidal Support to you guys… so I don’t know. 
We’ll just have to make do with it then.

Thanks

![IMG_4046](https://github.com/user-attachments/assets/0a895f29-8991-4c60-9f93-d934cd2b578d)
